private/manageFishData.py

```diff
@@ -244,6 +262,9 @@ def lookup_weather_by_name(name):
 
 
 def lookup_fishing_spot_by_name(name):
+    #if more than 1 instance is available in raw data, take the first one
```

Comment:
This can't actually happen, can it?  I was pretty certain the placename values only appear once.

Reply:
Thsi happens for me with A LOT of fish. As i parsed all fish with all data from ff14angler.
Since your version only include big fish (where it can't really happen i think) it would not be needed, but make things way easier for me.

Reply:
I see.  It's sorta a side-effect of normalization.  Ideally, you'd retain all of the data and only filter it down at the presentation layer, but like you say, for big fish, they only have a single location.